webapp/app/forms/steps/lotse/pauschbetrag.py

```diff
@@ -24,7 +24,9 @@ def calculate_pauschbetrag(has_pflegegrad=False, disability_degree=None, has_mer
     """
     if has_pflegegrad or has_merkzeichen_bl or has_merkzeichen_tbl or has_merkzeichen_h:
         return 7400
-
+    
+    if disability_degree == None:
+        return 0
```

Comment:
How is this ever possible if you only call it with `int(self.stored_data.get('person_b_disability_degree', 0))`?

Reply:
Dunno, but if i call this method just without parameters, the default value for the disability degree within the method is None. I thought before I change the default value, I rather react to None. What do you think?

Reply:
Mh, I see. I think we should use the same default value everywhere. Be it None or 0, I don't care.

Reply:
Did you change the default value? I didn't see.

Reply:
Yes i change it for the `self.stored_data.get('person_b_disability_degree', None)`